src/index.js

```diff
@@ -11,8 +11,7 @@ import store from './store';
 import { getMetadata, getStrings, getAbilities, getNeutralAbilities, getAbilityIds } from './actions';
 import App from './components/App';
 import constants from './components/constants';
-// import registerServiceWorker from './registerServiceWorker';
-import { unregister } from './registerServiceWorker';
+import registerServiceWorker from './registerServiceWorker';
```

Comment:
we previously had issues with the default CRA service worker.  not sure if that's still an issue

Reply:
nvm looks like you changed the path to another SW.  I would probably make a copy of that file and change the path there so we can enable the CRA service worker at some later point